docs/tutorial/write-your-resolvers.md

```diff
@@ -0,0 +1,125 @@
+---
+id: write-your-resolvers
+title: Write your resolvers
+sidebar_label: 5. Write your resolvers
+---
+
+[Previsouly](/docs/tutorial/create-server), we created our application instance with a [first iteration of our Schema](/docs/tutorial/create-server#recipes-manager-sdl-querygraphql), based on the SDL. Aside, we are going to bind some datas to our Schema, thank to the resolvers, which are in charge of make the bridge between the Schema and your application logic. 
+
+## What is a resolver?
+
+A resolver is a function _(which could be asynchronous)_ ties to a field of your Schema. It will be in charge of providing data accordingly to the fields or objects it is attached on.
+
+### How to attach a resolver to a Schema with Tartiflette?
+
+With tartiflette, it is very straighforward to attach a resolver function to the Schema.
+
+We provide a decorator named `Resolver` which will attach your function resolution to the Schema field specified. 
+
+```python
+from tartiflette import Resolver
+
+@Resolver("Query.hello")
+async def resolver_fieldname(parent, args, ctx, info):
+    return {
+        "foo": "bar"
+    }
+```
+
+In this example, the method "resolver_fieldname" will be executed each time when the field `hello` will be called. If you want to know more about the Resolver part, we suggest you to take a look of [the Resolver API documentation](/docs/api/resolver).
```

Comment:
In this example, the function `resolver_fieldname` will be executed each time the field `hello` is called.

     If you want to know more about the Resolver part, we suggest taking a look at [the Resolver API documentation]